lib/tasks/buildThemes.js

```diff
@@ -128,19 +164,51 @@ export default async function({
 				log.verbose(`Available sap.ui.core themes: ${availableThemes.join(", ")}`);
 			}
 		}
-		allResources = allResources.filter(isAvailable);
+		themeResources = themeResources.filter(isAvailable);
 	}
 
-	const processedResources = await themeBuilder({
-		resources: allResources,
-		fs: fsInterface(combo),
-		options: {
-			compress,
-			cssVariables: !!cssVariables
+	let buildTheme;
+	if (useWorkers) {
+		if (!taskUtil) {
+			// TaskUtil is required for worker support
+			throw new Error(`buildThemes: Option 'useWorkers' requires a taskUtil instance to be provided`);
 		}
-	});
+		buildTheme = buildThemeInWorker;
+	} else {
+		// Do not use workerpool
+		buildTheme = (await import("../processors/themeBuilderWorker.js")).default;
+	}
+
+	// All the relevant resources need to be made available for the less parser within the workerpool
+	const allResources = await combo.byGlob("/resources/**/{*.less,.theming,img/**,img-RTL/**}");
+	const makeTransferableResource = async (resourceCollection) =>
+		Promise.all(
+			resourceCollection.map(async (res) => ({
+				transferable: await res.getBuffer(), // Buffer -> Uint8Array (transferable object)
+				path: res.getPath()
+			}))
+		);
+	let allResourcesTransferable;
+	if (allResources) {
```

Comment:
How can `allResources` become falsy? `byGlob` should always return an Array, even when it's empty.

Reply:
I guess, it's because of the tests. I've been missled by the stubs. Thanks!